kotskinds/apis/kots/v1beta1/config_types.go

```diff
@@ -56,12 +56,22 @@ type ConfigItem struct {
 	CountByGroup  map[string]int         `json:"countByGroup,omitempty"`
 	Templates     []RepeatTemplate       `json:"templates,omitempty"`
 	ValuesByGroup ValuesByGroup          `json:"valuesByGroup,omitempty"`
+	Validation    *ConfigItemValidator   `json:"validation,omitempty"`
 	// Props       map[string]interface{} `json:"props,omitempty"`
 	// DefaultCmd  *ConfigItemCmd         `json:"default_cmd,omitempty"`
 	// ValueCmd    *ConfigItemCmd         `json:"value_cmd,omitempty"`
 	// DataCmd     *ConfigItemCmd         `json:"data_cmd,omitempty"`
 }
 
+type ConfigItemValidator struct {
```

Comment:
for naming consistency:
```suggestion
type ConfigItemValidation struct {
```